### PR TITLE
Feature/#81 avoid empty label entries

### LINKED ISSUE #81: Make sure LabelProcessor removes entries without values.
Avoid empty language entries in LabelType.

### PR DESCRIPTION
**Pull request by DivineThreepwood**
### :scroll: Description

Changes proposed in this pull request:
* auto remove empty label entries.

### :white_check_mark: Checklist:

- [x] Created tests which fail without the change (if possible)

### CODE REVIEW

File: module/extension/type/processing/src/main/java/org/openbase/jul/extension/type/processing/LabelProcessor.kt [hunk 1]
```diff
@@ -20,34 +26,19 @@
...
+object LabelProcessor {
 
-    public static final String LANGUAGE_CODE_TECHNICAL = "technical";
-    public static final Label UNKNOWN_LABEL = LabelProcessor.addLabel(Label.newBuilder(), Locale.ENGLISH, "?").build();
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(LabelProcessor.class);
+    const val LANGUAGE_CODE_TECHNICAL = "technical"
 
+    @JvmField
+    val UNKNOWN_LABEL = addLabel(LabelType.Label.newBuilder(), Locale.ENGLISH, "?").build()
```
Comment (pLeminoq) [inline module/extension/type/processing/src/main/java/org/openbase/jul/extension/type/processing/LabelProcessor.kt]: maybe use generateLabelBuilder or buildLabel instead of addLabel

File: module/extension/type/processing/src/test/java/org/openbase/jul/extension/type/processing/LabelProcessorTest.kt [hunk 2]
```diff
@@ -49,6 +49,26 @@ class LabelProcessorTest {
         Assertions.assertEquals("Living Cei Ling Lamp", LabelProcessor.format("Living   CeiLing      Lamp"))
     }
 
+    @Test
+    fun formatLabel() {
+        println("formatLabel")
```
Comment (pLeminoq) [inline module/extension/type/processing/src/test/java/org/openbase/jul/extension/type/processing/LabelProcessorTest.kt]: Maybe rename to something more specific such as `test format removes language with empty labels`

**Review by pLeminoq — APPROVED** [review]
Looks good. Just 2 minor suggestions you can have a look at.

**Review by pLeminoq — APPROVED** [review]

**Review by pLeminoq — APPROVED** [review]
